wordpress-mobile/WordPress-iOS
# Fix Blogging Reminders UI on iPad

Fixes #NA

This PR fixes the Blogging Reminders UI on iPad: previously it was presented as a popover, now it's using the `formSheet` modal presentation style.

[WordPressUI PR](https://github.com/wordpress-mobile/WordPressUI-iOS/pull/96)

To test:
Test on iPad:
- build/run and go to My Site if you're not already there
- tap on "get started" in the "Set your blogging goals" card
- Make sure that the "set your blogging goals" screen appears in the center of the page (see screenshots)
- tap "Set goals" and then "Next" and make sure the UI looks consistent with the screenshots below
Test on iPhone
- Repeat the steps and make sure that the screens are presented as a bottom sheet with variable height (see screenshots)
---
**iPad**
first screen | second screen | third screen
-- | -- | --
<img width="300" src="https://user-images.githubusercontent.com/34376330/121954300-382e3080-cd24-11eb-97a8-2f88a35ea779.png"> | <img width="300" src="https://user-images.githubusercontent.com/34376330/121954367-48dea680-cd24-11eb-9981-d2cde71e7738.png"> | <img width="300" src="https://user-images.githubusercontent.com/34376330/121954417-572cc280-cd24-11eb-89e7-524f1b0e58e7.png">

---


**iPhone**
first screen | second screen | third screen
-- | -- | --
<img height="400" src="https://user-images.githubusercontent.com/34376330/121955243-53e60680-cd25-11eb-9c7d-47c7c660b762.png"> | <img height="400" src="https://user-images.githubusercontent.com/34376330/121955296-67916d00-cd25-11eb-87a3-893bf8e6dca1.png"> | <img height="400" src="https://user-images.githubusercontent.com/34376330/121955376-7d9f2d80-cd25-11eb-9e44-0d79f987574a.png">
---

Known issues:
- Page transitions on both iPhone and iPad are still being worked on, as a result, there will be some weird animation/transition between screens
- View elements are still being worked on, so the positioning (e.g. of the button in the second screen) will be fixed in future PRs.

## Regression Notes
1. Potential unintended areas of impact
This PR changes the class `BottomSheetViewController` in `WordPressUI`, but the changes are transparent to the existing code, so there should not be any unintended impact.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Smoke tested other parts that use `BottomSheetViewController` on iPad (e.g filter on Reader or user profile on Notifications)

3. What automated tests I added (or what prevented me from doing so)
None
PR submission checklist:
